VAS.Services/Controller/TaggingController.cs

```
@@ -95,7 +95,7 @@ public override void Stop ()
 		public override void SetViewModel (IViewModel viewModel)
 		{
 			project = (ProjectVM)(viewModel as dynamic);
-			VideoPlayer = (VideoPlayerVM)(viewModel as dynamic);
+			VideoPlayer = (viewModel as IVideoPlayerVM)?.VideoPlayer;
```

Actually this is changed in VAS-46 task, there cannot be null and should throw an exception